packages/components/inputs/rich-text-input/src/editor.tsx

```diff
@@ -56,23 +106,21 @@ const Editor = (props) => {
         ) {
           toggle();
         }
+        const refObj: TRichtTextEditorBodyRef = {
+          containerRef: ref,
+          registerContentNode,
+        };
         return (
           <Constraints.Horizontal max={props.horizontalConstraint}>
             <Stack scale="xs" alignItems="flex-end">
-              <EditorWrapper
-                isDisabled={props.isDisabled}
-                isReadOnly={props.isReadOnly}
```

Comment:
this prop turned out to be unused

Reply:
Actually I think we need to fix that, as according to the localized-richt-text-input we do use it

https://github.com/commercetools/ui-kit/blob/c485b1899cc57a053f924008d7ec688534b08330/packages/components/inputs/localized-rich-text-input/src/editor.styles.js#L56

Can you fix it for this component as well? Thanks